### DISCUSSION TITLE
Next Route Visualizer

### DISCUSSION THREAD

**Discussion by DiiiaZoTe** [vercel/next.js#46802] [Show and tell]
<div align="center">
  <h1>next-route-visualizer</h1>
  <p>I created a component/dashboard to visualize your Next.js app directory routes.</p>
  <a href="https://next-route-visualizer-eufp7fbs4-diiiazote.vercel.app">
    <b>Demo website</b> 
  </a>
  | 
    <a href="https://www.npmjs.com/package/next-route-visualizer">
    <b>NPM package</b> 
  </a>
  | 
    <a href="https://github.com/DiiiaZoTe/next-route-visualizer">
    <b>Github</b> 
  </a>
</div>

<br/>

<div align="center">
  <img src="https://next-route-visualizer.vercel.app/next-route-visualizer-preview.png" alt="Preview Next Route Visualizer" width="600"/>
</div>

<br/>

I thought this could help some people that are starting with Next or for others... I know I will be using this for project that get a little bigger to make sure I'm not missing anything 👍

**Comment by moskalakamil** [comment]
:D
